frontend/src/metabase/admin/people/containers/UserSuccessModal.jsx

```diff
@@ -82,3 +83,24 @@ const PasswordSuccess = ({ user, temporaryPassword }) => (
     </div>
   </div>
 );
+
+export default _.compose(
+  User.load({
+    id: (state, props) => props.params.userId,
+  }),
+  connect(
+    (state, props) => ({
+      temporaryPassword: getUserTemporaryPassword(state, {
+        userId: props.params.userId,
+      }),
+      isSsoConfigured:
```

Comment:
Can you move this to `MetabaseSettings` instead? There are some sso-related methods already https://github.com/metabase/metabase/blob/ee25189dd8073dc7e4a51b0a0ab15fcade3bf7ce/frontend/src/metabase/lib/settings.ts#L182. Could you use them instead of the loop?